master/internal/api_tensorboard.go

```diff
@@ -112,8 +105,8 @@ func (a *apiServer) GetTensorboard(
 		return nil, err
 	}
 
-	if ok, err := command.AuthZProvider.Get().CanGetNSC(
-		ctx, *curUser, model.UserID(resp.Tensorboard.UserId), command.PlaceHolderWorkspace); err != nil {
+	if ok, err := command.AuthZProvider.Get().CanGetTensorboard(
```

Comment:
should we add tensorboards to these tests as well https://github.com/hamidzr/determined/blob/2e16fc41f2a98a7afb8a124f1ad92b8d43c169e5/master/internal/api_ntsc_intg_test.go#L78 ? we could add them in a separate pr

Reply:
I was going to add them to a separate PR - I also was going to rename NSC to NTSC to include tensorboards, so I can do a follow up PR so this one doesn't get too large